osu.Game/Overlays/BeatmapSet/Info.cs

```diff
@@ -68,7 +66,10 @@ public Info()
                             Child = new Container
                             {
                                 RelativeSizeAxes = Axes.Both,
-                                Child = new MetadataSection("Description"),
+                                Child = description = new MetadataSection(MetadataType.Description, transition_duration)
+                                {
+                                    HeaderMargin = new MarginPadding { Top = 20 },
```

Comment:
maybe adding a small nested class that does these stuff (setting transition duration, header margin and colour), I think it'll look better than duplicating the lines